bunjs/templates.yaml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/agentuity/templates/main/templates.schema.json
-- name: "OpenAI SDK for Typescript"
```

Comment:
Standardizing names + description across frameworks and languages; concise Python version was better IMO so going with that to avoid repetition.